torch/csrc/jit/frontend/string_to_type.cpp

```diff
@@ -6,6 +6,15 @@ using namespace c10;
 const std::unordered_map<std::string, TypePtr>& string_to_type_lut() {
   static std::unordered_map<std::string, TypePtr> map = {
       {"Tensor", TensorType::get()},
+      {"LongTensor", TensorType::get()},
```

Comment:
I think we should leave a comment here explaining why `dtype` is kind of abandoned here.

Reply:
done